src/app.js

```diff
@@ -1,5 +1,218 @@
 const express = require('express');
+const strings = require('./lib/strings.js');
+const numbers = require('./lib/numbers.js');
+const bool = require('./lib/booleans.js');
+const arr = require('./lib/arrays.js');
 
 const app = express();
 
+app.use(express.json()); // FOR POST
+// app.use(bodyParser.json());
+
+app.get('/strings/hello/:basename', (req, res) => {
+  const word = strings.sayHello(req.params.basename);
+
+  res.json({ result: `${word}` }).sendStatus(200);
```

Comment:
This will probably work. However, I would suggest using .status() instead of .sendStatus().

sendStatus sets and sends the status.

This means that if you want to both set a status and send a body you have to use status. For example, you want to set an error status, and send a body with a JSON that explains why the error occurred, you first have to set the status (using status), and then send the JSON (using send). If you had already set the status with sendStatus, it is no longer possible to send the JSON, because you already used a form of sent.

Reply:
I started off with .sendStatus which was working fine initially but later on during the project it backstabbed me so I switched to .status lol. I later understood that .sendStatus() closes the connection and you cannot send or .json after it.